index.js

```diff
@@ -20,17 +21,60 @@ const releaseOldLockedPortsIntervalMs = 1000 * 15;
 // Lazily create interval on first use
 let interval;
 
-const getAvailablePort = options => new Promise((resolve, reject) => {
-	const server = net.createServer();
-	server.unref();
-	server.on('error', reject);
-	server.listen(options, () => {
-		const {port} = server.address();
-		server.close(() => {
-			resolve(port);
+const getHosts = () => {
+	const interfaces = os.networkInterfaces();
+	const results = [];
+
+	for (const _interface of Object.values(interfaces)) {
+		for (const config of _interface) {
+			results.push(config.address);
+		}
+	}
+
+	// Add undefined value for createServer function to use default host,
+	// and default IPv4 host in case createServer defaults to IPv6.
+	return results.concat(undefined, '0.0.0.0');
```

Comment:
`0.0.0.0` could potentially be twice in the `results` array. I would recommend using a `Set` instead.